apps/website/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "website",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "TAMAGUI_TARGET='web' next dev",
```

Comment:
Fails on

```shell
felicio@felicios-mbp website % yarn dev
yarn run v1.22.19
$ TAMAGUI_TARGET='web' next dev
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
(node:95996) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)

Tamagui built config and components:

  Config     ./.tamagui/tamagui.config.cjs
  Components ./.tamagui/statusimcomponents-components.config.cjs

Note: We've moved @tamagui/theme-base over to @tamagui/themes. They are much improved and easier to use. If you really want to keep this package, copy and paste the contents into your app as it will be removed eventually.
Note: We've moved @tamagui/theme-base over to @tamagui/themes. They are much improved and easier to use. If you really want to keep this package, copy and paste the contents into your app as it will be removed eventually.
Tamagui Webpack Loader Error:
   createCSSVariable expected string: undefined
Error: createCSSVariable expected string: undefined
    at createCSSVariable (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/core-node/dist/index.js:5842:17)
    at variableToCSS (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/core-node/dist/index.js:10687:63)
    at /Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/core-node/dist/index.js:11140:78
    at createTamagui (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/core-node/dist/index.js:11210:9)
    at loadTamagui (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/static/dist/cjs/extractor/loadTamagui.js:155:40)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.load [as loadTamagui] (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/static/dist/cjs/extractor/createExtractor.js:92:42)
    at async extractToClassNames (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/static/dist/cjs/extractor/extractToClassNames.js:78:3)
    at async Object.loader2 (/Users/felicio/Projects/status-im/status-web/node_modules/tamagui-loader/dist/cjs/loader.js:62:23)

Missing theme, an error occurred when importing your config
Got config: [Function (anonymous)] {
  displayName: 'ProxyWorm - Check excludeReactNativeWebExports',
  _isProxyWorm: true
}
Looking for theme: displayName

✨  Done in 4.91s.
```

Reply:
Using `DEBUG=tamagui` prints

```shell
Tamagui failed loading the pre-built tamagui.config.ts
  
  The method or property expo-modules-core.requireNativeViewManager is not available on web, are you sure you've linked all the native dependencies properly?
  Error: The method or property expo-modules-core.requireNativeViewManager is not available on web, are you sure you've linked all the native dependencies properly?
    at requireNativeViewManager (/Users/felicio/Projects/status-im/status-web/apps/website/.tamagui/statusimcomponents-components.config.cjs:38815:9)
    at ../../node_modules/expo-blur/build/BlurView.js (/Users/felicio/Projects/status-im/status-web/apps/website/.tamagui/statusimcomponents-components.config.cjs:39088:22)
    at __init (/Users/felicio/Projects/status-im/status-web/apps/website/.tamagui/statusimcomponents-components.config.cjs:10:59)
    at ../../node_modules/expo-blur/build/index.js (/Users/felicio/Projects/status-im/status-web/apps/website/.tamagui/statusimcomponents-components.config.cjs:39101:5)
    at __init (/Users/felicio/Projects/status-im/status-web/apps/website/.tamagui/statusimcomponents-components.config.cjs:10:59)
    at Module.<anonymous> (/Users/felicio/Projects/status-im/status-web/apps/website/.tamagui/statusimcomponents-components.config.cjs:39927:11)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at Module.require (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/static/dist/cjs/require.js:56:22)
    at require (node:internal/modules/cjs/helpers:103:18)
    at attemptLoad (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/static/dist/cjs/extractor/loadTamagui.js:341:34)
    at /Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/static/dist/cjs/extractor/loadTamagui.js:349:21
    at Array.flatMap (<anonymous>)
    at loadComponents (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/static/dist/cjs/extractor/loadTamagui.js:315:36)
    at loadTamagui (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/static/dist/cjs/extractor/loadTamagui.js:126:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.load [as loadTamagui] (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/static/dist/cjs/extractor/createExtractor.js:92:42)
    at async extractToClassNames (/Users/felicio/Projects/status-im/status-web/node_modules/@tamagui/static/dist/cjs/extractor/extractToClassNames.js:78:3)
    at async Object.loader2 (/Users/felicio/Projects/status-im/status-web/node_modules/tamagui-loader/dist/cjs/loader.js:62:23)
  
    You can see if it loads in the node repl:
  
    require("./.tamagui/statusimcomponents-components.config.cjs").default
```

Reply:
Relates to

- https://github.com/tamagui/tamagui/issues/846
- https://github.com/tamagui/tamagui/issues/638
- https://github.com/tamagui/tamagui/issues/273

Reply:
Addressed at fb4f97d4e05797a9ff7b3b02226a69a9c08a695e

Reply:
Curious if this would be fixed by doing the platform re-export in the project...

Reply:
`expo-blur` is causing this, right @felicio?

Reply:
Should be fixed in https://github.com/tamagui/tamagui/releases/tag/v1.28.0 by https://github.com/tamagui/tamagui/pull/1168/commits/aa28bebd7bdc12baa3512f5cb24d00fd5097f953.